specification/2.0/schema/material.schema.json

```diff
@@ -5,28 +5,79 @@
+            "default" : [1,1,1,1],
+            "gltf_detailedDescription" : "The RGBA components of the base color (RGB) of the material. The fourth component (A) is the opacity of the material."
+        },
+         "baseColorTexture" : {
+            "extends" : "number",
+            "description" : "The index of the base color texture."
+        },
+        "baseColorTexCoord" : {
+            "extends" : "number",
+            "description" : "The set index of the base color texture's TEXCOORD attribute to be used for texture coordinate mapping."
+        },
+        "metallicFactor" : {
+            "type" : "number",
+            "description" : "The metallic-ness the material (1 for metals, 0 for non-metals).",
+            "default" : 1
+        },
+        "roughnessFactor" : {
+            "type" : "number",
+            "description" : "The perceptual roughness of the material surface (1 for rough, 0 for smooth)..",
+            "default" : 1,
+            "minimum" : 0,
+            "maximum" : 1
+        },
+        "metallicRoughnessTexture" : {
+            "extends" : "number",
+            "description" : "The index of the metallic-roughness texture."
+        },
+        "metallicRoughnessTexCoord" : {
+            "extends" : "number",
+            "description" : "The set index of the metallic-roughness texture's TEXCOORD attribute to be used for texture coordinate mapping."
+        },
+        "normalTexture" : {
+            "extends" : "number",
+            "description" : "The index the texture containing the normal map."
+        },
+        "normalTexCoord" : {
+            "extends" : "number",
+            "description" : "The set index of normalTexture's TEXCOORD attribute to be used for texture coordinate mapping"
+        },
+        "normalScale" : {
+            "extends" : "number",
+            "description" : "The scalar multiplier applied to each normal vector of the texture."
+        },
+        "occlusionTexture" : {
+            "extends" : "number",
+            "description" : "The index the texture containing the occlusion map."
+        },
+        "occlusionTexCoord" : {
+            "extends" : "number",
+            "description" : "The set index of occlusionTexture's TEXCOORD attribute to be used for texture coordinate mapping"
+        },
+        "occlusionStrength" : {
+            "extends" : "number",
+            "description" : "A scalar multiplier controlling the amount of occlusion applied. A value of 0 means no occlusion; 1.0 means full occlusion."
+        },
+        "emmissionTexture" : {
+            "extends" : "number",
+            "description" : "The index the texture containing the emission map."
+        },
+        "emmissionTexCoord" : {
+            "extends" : "number",
+            "description" : "The set index of emmissionTexture's TEXCOORD attribute to be used for texture coordinate mapping"
+        },
+        "emmissionStrength" : {
```

Comment:
Yup went back and forth between using factor for consistency vs trying to use something that explains what it is :). Just thought I will propose an alternative in case people like it. Since all of them are multipliers factor works well to and will update it to that,